joint_limits:
    front_left_wheel_joint:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      #max_velocity: 2.62
      #min_velocity: -2.62
      max_velocity: 12.57 #in radians
      min_velocity: -12.57
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: false
      max_effort: 0
      min_effort: 0
    
    front_right_wheel_joint:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      #max_velocity: 2.62
      #min_velocity: -2.62
      max_velocity: 12.57
      min_velocity: -12.57
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: false
      max_effort: 0
      min_effort: 0

    rear_left_wheel_joint:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      #max_velocity: 2.62
      #min_velocity: -2.62
      max_velocity: 12.57
      min_velocity: -12.57
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: false
      max_effort: 0
      min_effort: 0
      
    rear_right_wheel_joint:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      #max_velocity: 2.62
      #min_velocity: -2.62
      max_velocity: 12.57
      min_velocity: -12.57
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: false
      max_effort: 0
      min_effort: 0
